src/cpp/session/modules/automation/SessionAutomation.R

```diff
@@ -448,9 +481,6 @@
    .rs.setVar("automation.client", client)
    .rs.setVar("automation.sessionId", sessionId)
    
-   # Navigate the page to RStudio Server.
-   client$Page.navigate("http://localhost:8787")
```

Comment:
No longer necessary as we now point the browser directly at the running RStudio Server instance.